experiments/golden-results/vct-summary.txt

```diff
@@ -177,3 +172,13 @@ Occurs: 1 times
 +===========================GNAT BUG DETECTED==============================+
 | GNU Ada (ada2goto) Assert_Failure sinfo.adb:3253                         |
 Error detected at REDACTED
+--
+Occurs: 1 times
++===========================GNAT BUG DETECTED==============================+
+| GNU Ada (ada2goto) exception ureal_to_binary.adb:258                     |
```

Comment:
Any idea why we're getting extra crashes?

Reply:
This is within system.ads and is caused by having a negative exponent value. Unsure as to why is is now being reported.

Reply:
OK.